src/dispatcher.js

```diff
@@ -42,9 +43,9 @@ function buildGenerations( stores, actionType ) {
 }
 
 function processGeneration( generation, action ) {
-	generation.map( ( store ) => {
+	generation.map( store => {
 		const data = Object.assign( {
-			deps: _.pick( this.stores, store.waitFor )
+			deps: _.pick( this.stores, store.waitFor ) // eslint-disable-line
```

Comment:
we usually want to include which rule was being overridden here (i.e. - `// eslint-disable-line {rulename}`